protocol/msgpack/unaggregated_iterator.go

```diff
@@ -40,18 +40,18 @@ const (
 type unaggregatedIterator struct {
 	iteratorBase
 
-	ignoreHigherVersion bool
-	largeFloatsSize     int
-	largeFloatsPool     pool.FloatsPool
-	iteratorPool        UnaggregatedIteratorPool
+	largeFloatsSize int
+	largeFloatsPool pool.FloatsPool
+	iteratorPool    UnaggregatedIteratorPool
 
-	closed             bool
-	metric             unaggregated.MetricUnion
-	policiesList       policy.PoliciesList
-	id                 id.RawID
-	timerValues        []float64
-	cachedPolicies     [][]policy.Policy
-	cachedPoliciesList policy.PoliciesList
+	ignoreHigherVersion bool
```

Comment:
Can we move `ignoreHigherVersion` to be grouped together with the three fields above so all four are immutable fields?